deltas/engine/prefabs/test.prefab

```diff
@@ -0,0 +1,5 @@
+{
+    "RigidBody": {
+        "kinematic": false
```

Comment:
Should this file still be here?

Reply:
I don't think this is required, so will remove it.